source/sass/blocks/banner.scss

```
@@ -0,0 +1,132 @@
+.banners-big .banners-big__top-text {
+    display: block;
+    color: #000000;
+    font-size: 47px;
+    font-weight: 900;
+    line-height: 59px;
+    letter-spacing: -0.02em;
+    opacity: 1;
+
+    @media (max-width: 743px) {
+        font-size: 27px;
+        font-weight: 900;
+        line-height: 34px;
+        letter-spacing: -0.02em;
+    }
+}
+
+.banners-big .banners-big__text-block {
+    color: #000000;
+    font-family: 'CeraKFC-400';
+    font-weight: 400;
+    opacity: 1;
+    font-size: 16px;
+    line-height: 20px;
+}
+
+.banners-big__text--wide .banners-big__title {
+    color: var(--theme-base-color);
+    font-size: 135px;
+    font-weight: 900;
+    line-height: 170px;
+    letter-spacing: -0.04em;
+
+    @media (max-width: 1200px) {
+        font-size: 110px;
+        font-weight: 900;
+        line-height: 105px;
+        letter-spacing: -0.04em;
+        margin-bottom: 20px;
+        width: 65%;
+    }
+
+    @media (max-width: 744px) {
+        font-size: 74px;
+        font-weight: 900;
+        line-height: 70px;
+        letter-spacing: -0.04em;
+    }
+
+    @media (min-width: 1440px) {
+        width: 1000px;
+    }
+}
+
+.banners-big__text--wide.banners-big__text--center .banners-big__text-wrapper {
+    width: 614px;
+}
+
+@media (min-width: 1100px) {
+    .banners-img-with-text-list .grid-list--items-3 {
+        width: 1050px;
+        grid-template-columns: repeat(auto-fill, minmax(325px, 1fr));
+        justify-content: center;
+        // margin: 0 auto;
+    }
+}
+
+.banners-img-with-text-list .grid-list__item {
+    max-width: 335px;
+    min-height: 325px;
+}
+
+.banners-img-with-text-list .rounded-4 {
+    border-radius: 20px;
+}
+
+.banners-img-with-text-list__item-image.banners-img-with-text-list__item-image--short {
+    padding-top: 100%;
+}
+
+.banners-big__wrapper::before {
+    content: "";
+    position: absolute;
+    background-image: url("../img/hand-left.svg");
+    background-repeat: no-repeat;
+    width: 234px;
+    height: 276px;
+    top: 180px;
+    left: 0;
+    z-index: 2;
+}
+
+.banners-big__wrapper::after {
+    content: "";
+    position: absolute;
+    background-image: url("../img/hand-right.svg");
+    background-repeat: no-repeat;
+    width: 175px;
+    height: 118px;
+    top: 180px;
+    right: 0;
+    z-index: 2;
+}
+
+// @media (max-width: 1200px) {
+//     .banners-big__wrapper::after {
+//         background-size: 153px 103px;
+//     }
+
+//     .banners-big__wrapper::before {
+//         height: 241px;
+//         width: 152px;
+//     }
+
+// }
+
+// .banners-img-with-text-list  {
+//   position: relative;
+// }
+
+.banners-img-with-text-list::before {
+    content: "";
+    position: absolute;
+    background-image: url("../img/hand-bottom.svg");
+    background-repeat: no-repeat;
+    width: 300px;
+    height: 220px;
+    top: -141px;
+    right: 550px;
```

Думаю, эта рука должна быть все-таки по центру

```suggestion
    left: calc(50% - 100px);
```